operators/image_vec_rep_resnet/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
```

Comment:
a questions about this file

if I understand correctly, each operator will require a `pyproejct.toml` file right?
and current we have a folder at source of Feluda called `operators`, inside which we have `operators/image_vec_rep_resnet` and inside it there is a `.toml` file and a `src/image_vec_rep_resnet/` folder -- will we have to maintain this structure for each operator? or can we have all operators at one place?

Reply:
We have to maintain that structure for each operator since we are releasing each package separately. Each operator also has different dependencies so it makes sense to make different .toml for different operators.

Reply:
I see understood!

so a .toml file will fact as a requirements file for each operator and will allow the user to install each operator separately perfect!